token/bearer.go

```diff
@@ -111,10 +144,12 @@ func (b *BearerToken) SignToken(key *ecdsa.PrivateKey) error {
 	return nil
 }
 
-func (b BearerToken) Signature() *signature.Signature {
-	return signature.NewFromV2(b.token.GetSignature())
+// Signature returns signature value of bearer token.
+func (b BearerToken) Signature() signature.Signature {
```

Comment:
I guess `IsValidSignature() bool`  is enough.

Reply:
True. `Signature()` is used right now in neofs-node ([link](https://github.com/nspcc-dev/neofs-node/blob/923db59149c4d97083ed656f8670e5e156925451/pkg/services/object/acl/acl.go#L230), but it can be replaced with `Issuer()` method.